course_inventory/inventory.py

```diff
@@ -182,6 +182,31 @@ def gather_course_data_from_api(account_id: int, term_ids: Sequence[int]) -> pd.
     return course_df
 
 
+def gather_account_data_from_api(account_ids: Sequence[int]) -> pd.DataFrame:
```

Comment:
Can you help me understand why this is necessary? I am using similar patterns throughout this file, so if I'm doing something wrong or unsafe, I'd like to know why. Account IDs when they are found on their own, in my view, should be an integer, since they need to be integers to be valid. Yes, I'm only using this as a string here, but relaxing the type hint doesn't make sense to me, since only valid IDs (integers) will help build a valid URL. I'm not seeing why every function/method needs to thoroughly validate its inputs; I don't think you're doing this in `extract.py`.

Reply:
As we discussed in Slack, this probably isn't a big deal.  This is an internal function, not part of a public API.  Using `str()` on the values in the function probably handles 99% of the problems with receiving the wrong data type.  The only place this function is called uses code that would seem to always return a sequence.